src/scripts/public/dom-loader.js

```diff
@@ -131,6 +141,75 @@ const domReady = () => {
             body.style.paddingTop = 0;
         }
 
+        /**
+         * Set up external icons for links.
+         */
+
+        if ('1' === window.site.externalLinkIcons) {
+            const links = document.querySelectorAll('#post-content a');
+
+            if (links) {
+                links.forEach((link) => {
+                    const href = link.getAttribute('href');
+
+                    // Check if the link is an anchor link or a relative link
+                    if (
+                        href.startsWith('#') ||
+                        href.startsWith('/') ||
+                        href.startsWith('./') ||
+                        href.startsWith('../')
+                    ) {
+                        return;
+                    }
+
+                    // Get the current domain
+                    const currentDomain = window.location.hostname;
+
+                    // Extract the domain from the link's href
+                    const linkDomain = href.match(
+                        /^(?:https?:)?(?:\/\/)?([^\/\?]+)/i
+                    )[1];
```

Comment:
I tried enabling this on the EIBC site locally and I was getting an error here: `Uncaught TypeError: Cannot read properties of null (reading '1')` so we might need to check if linkDomain has an element 1 before using it here.

Reply:
Yeah, I should test for that for sure. Named anchor links will fail.